docker/api/container.py

```diff
@@ -320,7 +321,7 @@ def start(self, container, binds=None, port_bindings=None, lxc_conf=None,
               dns=None, dns_search=None, volumes_from=None, network_mode=None,
               restart_policy=None, cap_add=None, cap_drop=None, devices=None,
               extra_hosts=None, read_only=None, pid_mode=None, ipc_mode=None,
-              security_opt=None, ulimits=None):
+              security_opt=None, ulimits=None, healthcheck=None):
```

Comment:
This part is incorrect - we do not want to update the `start` method with new parameters. Especially in this case, healthcheck is part of the `ContainerConfig`, not the `HostConfig`, so it's not valid to have here.